templates/agent/src/index.ts

```
@@ -6,15 +6,14 @@ import {
-import { registerSkill as coolSkill } from "./handlers/cool.js";
-import { registerSkill as infoSkill } from "./handlers/info.js";
-import { registerSkill as registerSkill } from "./handlers/register.js";
-import { registerSkill as renewSkill } from "./handlers/renew.js";
-import { registerSkill as paySkill } from "./handlers/pay.js";
-import { registerSkill as resetSkill } from "./handlers/reset.js";
-import { registerSkill as tokenSkill } from "./handlers/token.js";
-import { registerSkill as gameSkill } from "./handlers/game.js";
+import { registerSkill as checkSkill } from "./skills/check.js";
+import { registerSkill as coolSkill } from "./skills/cool.js";
+import { registerSkill as infoSkill } from "./skills/info.js";
+import { registerSkill as registerSkill } from "./skills/register.js";
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)